<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>user_register</class>
 <widget class="QDialog" name="user_register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Voiceprint Recognition</string>
  </property>
  <widget class="QLabel" name="register_Label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>220</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>32</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>用户注册</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="name_reminder_label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>111</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>请输入姓名：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameinput_lineedit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>your_name</string>
   </property>
  </widget>
  <widget class="QLabel" name="record_reminder_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>271</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="text">
    <string>点击[录音]按钮后

清晰读出0,1,2,3,4,5,6,7,8,9

重复三遍后点击[保存]</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="record_pushbutton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>480</y>
     <width>60</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>录音</string>
   </property>
  </widget>
  <widget class="QPushButton" name="back_pushbutton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>60</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
